scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/LiveJobs.java

```diff
@@ -264,7 +264,7 @@ boolean pauseJob(JobId jobId) {
 
             if (!updatedTasks.isEmpty()) {
                 jlogger.info(jobId, "has just been paused.");
-                dbManager.updateJobAndTasksState(job);
+                dbManager.updateJobAndTasksStatuses(job, updatedTasks, TaskStatus.PAUSED);
```

Comment:
does it make sense to analyse if the same strategy works when the job is resumed ?

Reply:
I  think i will redo this PR.